app-text/booksorg/additional-files/booksorg.rdef.in

```diff
@@ -0,0 +1,46 @@
+
+resource app_flags B_MULTIPLE_LAUNCH;
+
+resource app_version {
+	major  = @MAJOR@,
+	middle = @MIDDLE@,
+
```

Comment:
Hmm, I missed this one. It would be nice to insert a line for `minor`:
```diff
 resource app_version {
 	major  = @MAJOR@,
 	middle = @MIDDLE@,
+	minor  = @MINOR@,
 
 	variety = B_APPV_FINAL,
```